pylint/lint/pylinter.py

```diff
@@ -158,6 +158,11 @@ def _load_reporter_by_class(reporter_class: str) -> type:
         "bad-plugin-value",
         "Used when a bad value is used in 'load-plugins'.",
     ),
+    "E0014": (
+        "Incorrect setting encountered in top level configuration-section '%s' : '%s'",
+        "bad-configuration-option-value",
+        "Used when a top section value can't be parsed probably because it does not exists.",
```

Comment:
```suggestion
        "Used when a top section value can't be parsed because it does not belong to the top section "
        "or has an incorrect value type.",
```